tests/test_ir.py

```diff
@@ -524,7 +524,7 @@ def test_iteration_parallelism_2d(self, exprs, atomic, parallel):
         for i, e in enumerate(list(exprs)):
             exprs[i] = eval(e)
 
-        op = Operator(exprs, dle='advanced')
+        op = Operator(exprs, dle='openmp')
```

Comment:
wait, what?

Reply:
with `advanced` cache blocking dims appear which make it fail, 

># Use 'openmp' here instead of 'advanced' to disable loop blocking
line 584 for 3D, should be same here

Reply:
then it's a bug to be fixed

Reply:
It;s not a bug, that's what's needed for that test

Reply:
ahhhhhh sorry now I see what you mean!